hawkbit-sdk/hawkbit-sdk-dmf/src/main/java/org/eclipse/hawkbit/sdk/dmf/DmfController.java

```diff
@@ -83,10 +85,24 @@ public void stop() {
     }
 
     public void processUpdate(final EventTopic actionType, final DmfDownloadAndUpdateRequest updateRequest) {
-        updateHandler.getUpdateProcessor(this, actionType, updateRequest);
+        log.info("Processing update for {} actionId : {}", controllerId, updateRequest.getActionId());
+        updateHandler.getUpdateProcessor(this, actionType, updateRequest).run();
+        updateRequest.getSoftwareModules().forEach(sm -> {
```

Comment:
Since it is not said that every DMF user will update attributes after an update in this way this attribute update shall not be made here. 
The UpdateHandler is extending an endpoint. If someone wants to to that attribute update he could extend UpdateHandler overriding getUpdateProcessor by:
super.getUpdateProcesspr(controller, actionType, updateRequest);
updateRequest.getSoftwareModules().forEach(sm -> { ...)

Reply:
removed it.